scripts/ci-entrypoint.sh

```diff
@@ -212,6 +212,18 @@ if [[ -n "${TEST_CCM:-}" ]]; then
     fi
 
     export CCM_LOG_VERBOSITY="${CCM_LOG_VERBOSITY:-4}"
+    # Get CCM cluster CIDRs from cluster config template if not set
+    if [[ -z "${CCM_CLUSTER_CIDR:-}" ]]; then
+        curl -LO https://github.com/mikefarah/yq/releases/download/v4.30.3/yq_linux_amd64 && chmod +x yq_linux_amd64
+        while read -r CCM_CIDR; do
+            if [[ "${CCM_CLUSTER_CIDR:-}" == "" ]]; then
+                CCM_CLUSTER_CIDR="${CCM_CIDR}"
+            else
+                CCM_CLUSTER_CIDR="${CCM_CLUSTER_CIDR:-}\,${CCM_CIDR}"
+            fi
+        done <<< "$(./yq_linux_amd64 '.spec.clusterNetwork.pods.cidrBlocks' "${CLUSTER_TEMPLATE}" --no-doc|grep -v null | awk '{print $2}')"
```

Comment:
Why do we need to grep the template here, can't we just use kubectl to get the value? At this point `Cluster`  is an object on the management cluster

Reply:
You need to do this before line 191 so we're still pointing at the management cluster kubeconfig but then you should be able to do something like:

` "${KUBECTL}" get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks}'`

Reply:
@CecileRobertMichon Thank you for the advice. PTAL.